packages/presentation/lib/screens/login/login_bloc.dart

```diff
@@ -30,6 +30,7 @@ abstract class LoginBloc implements Bloc<BaseArguments, LoginData> {
       );
 
   GlobalKey<FormState> get formStateGlobalKey;
+  set formStateGlobalKey(GlobalKey<FormState> value);
```

Comment:
define the key in the way u define textcontroller

Reply:
Fixed.